source/utils/statsd-client.ts

```diff
@@ -0,0 +1,31 @@
+import { wrap } from 'underscore'
+import { hostname } from 'os'
+import { StatsD } from 'hot-shots'
+import StatsdClient from 'hot-shots'
+import { loadCustomConfiguration } from '../config'
+import { ConfigDefault } from '../config.default';
+
+const statsdConfig = loadCustomConfiguration().statsdConfig
+
+export function client(config: ConfigDefault): StatsD{
+    const client = new StatsdClient({
+        host: config.statsdConfig.host,
+        cacheDns: true,
+        telegraf: true,
+        errorHandler: (err: Error) => console.error(err.stack || err),
+        prefix: `${config.statsdConfig.prefix}.${config.statsdConfig.env}.system.${hostname()}_`,
+        globalTags: {}
+    })
+
+    if (statsdConfig.debug) {
+        StatsdClient.prototype.sendMessage =
```

Comment:
Is it possible to replace the method of the `client`, not of `StatsdClient.prototype`?

Reply:
didn't find it